civet.dev/cheatsheet.md

```diff
@@ -315,3 +315,39 @@ floor # / 2
 ```
 
 </div>
+
+## ASCII Symbols
+
+Here is a list of ASCII symbols and their various contextual meanings in Civet.
+(There are also more [Unicode symbols](/reference#unicode-operators).)
+
+| Symbol | Meanings |
+|--------|----------|
+| space | Function call |
+| `?` | Non-null test `x?`; non-null access `x?.y`, `x?y`, `x?(...)`, `x?[...]`; null coalescing `x ?? y` (`x ? y` with `coffeeBinaryExistential`); TypeScript optionals `(x?: number) => x`; if/then/else ternary `x ? y : z` |
```

Comment:
Optional types `number?`, `number??`, optional declarations `let x?: number`, optional check `if x? := foo()`

As an aside, `let x?` with no type annotation is currently left alone during compilation.

Reply:
I wonder what `let x?` should mean. I think most intuitively it would say "allow `x` to be unassigned on first access" but I'm not sure this is possible in TypeScript. Perhaps a helpful error message for now.